File: backend/Models/Comment.cs
using Microsoft.Identity.Client;

namespace backend.Models;

public class Comment
{
    public int? ID { get; set; }

    public int? ParentID { get; set; }

    public int? PostID { get; set; }

    public int? UserID { get; set; }

    public string? Content { get; set; }
}

File: frontend/src/types/Comment.ts
export default interface CommentModel {
  comment_id: number;
  parent_id: number | null;
  post_id: number;
  user_id: number;
  username: string;
  content: string;
  pfp_url: string;
  num_likes: number;
  liked: boolean;
}
